definitions/divorce/json/AuthorisationCaseEvent.json

```diff
@@ -1861,6 +1861,13 @@
     "UserRole": "caseworker-divorce-courtadmin_beta",
     "CRUD": "CRU"
   },
+  {
```

Comment:
Should this be going to prod?

Reply:
this will go in https://tools.hmcts.net/jira/browse/RDM-5463 so has to go too